components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/dto/TierDTO.java

```diff
@@ -23,6 +23,12 @@
   
   private String description = null;
   
+  public enum TierLevelEnum {
+     api,  application,  resource, 
```

Comment:
If you required you can have them via toLowerCase(). But ENUMs treated as constants which will be named in uppercase